### PR TITLE
Features/add linters

### PR DESCRIPTION
**Pull request by andiserg**
refactor code by linters. Add linters checking workflow to CI

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -52,6 +55,11 @@ func main() {
 	)
 	db, err := gorm.Open(postgres.Open(dsn), &gorm.Config{})
 
+	if err != nil {
+		log.Fatal(err)
```
Comment (andriihar) [inline main.go:59]: What will happen in log.Fatal() func?  

You can read about os.Exit(1) 

https://go.dev/play/p/pwfwPbQ8nnb

File: src/adapters/email.go [hunk 2]
```diff
@@ -2,33 +2,28 @@ package adapters
...
 
-func (ea EmailAdapter) CreateAuth() {
-	ea.Auth = smtp.PlainAuth("", ea.Username, ea.Password, "smtp.gmail.com")
-}
-
 func (ea EmailAdapter) Send(target string, rate float32) {
-	println(fmt.Sprintf("Send email from: %s; to: %s; rate: %f", ea.Username, target, rate))
-	/*
-		to := []string{target}
+	//println(fmt.Sprintf("Send email from: %s; to: %s; rate: %f", ea.Username, target, rate))
```
Comment (andriihar) [inline src/adapters/email.go]: Do we need this comment?

File: src/adapters/email.go [hunk 3]
```diff
@@ -2,33 +2,28 @@ package adapters
...
 
-		err := smtp.SendMail("smtp.gmail.com:587", ea.Auth, ea.Username, to, msg)
+	err := smtp.SendMail("smtp.gmail.com:587", ea.Auth, ea.Username, to, msg)
 
-		if err != nil {
-			log.Fatal(err)
-		}
-	*/
+	if err != nil {
+		log.Fatal(err)
```
Comment (andriihar) [inline src/adapters/email.go]: What will happen in log.Fatal() func?

You can read about os.Exit(1)

https://go.dev/play/p/pwfwPbQ8nnb

File: .github/workflows/linters.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+name: LintersCheck
+
+on:
+  push:
+    branches: ["project"]
```
Comment (AlwaysSayNo) [inline .github/workflows/linters.yaml]: I couldn't find "project" branch in your project. Maybe you need to use "main" instead?

File: .github/workflows/linters.yaml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.22'
+      - name: Build
+        run: go build -v ./...
+      - name: Test
+        run: go test -v ./...
```
Comment (AlwaysSayNo) [inline .github/workflows/linters.yaml]: I'm not sure, but I think you don't need to have build+test stage for linter action. Probably you could create a separate action for this.

**Review by AlwaysSayNo — CHANGES_REQUESTED** [review]

**Review by andriihar — APPROVED** [review]

**Comment by andriihar** [comment]
For the future, you can add linters you want to use and configure them.

**Pull request merged**